spec/system/links_spec.rb

```diff
@@ -1,19 +1,35 @@
 require 'rails_helper'
 
-RSpec.describe('Quick Links') do
+# Used for creating a factory bot link w/ admin user
+def make_link(**kwargs)
+  admin_user = User.find_by(email: 'admin@gmail.com')
+  FactoryBot.create(:link, user: admin_user, **kwargs)
+end
+
+# Used to sign in as admin
+def sign_in_admin
+  admin_user = User.find_by(email: 'admin@gmail.com')
+  sign_in(admin_user)
```

Comment:
Could use "sign_in admin_user" for consistency

Reply:
rubocop's fault